packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/dropdown/splitbutton.css

```diff
@@ -33,7 +33,8 @@
 		/* It's a text-less button and since the icon is positioned absolutely in such situation,
 		it must get some arbitrary min-width. */
 		min-width: unset;
-
+		border: none;
```

Comment:
This makes the focused "arrow button" lose border

![](https://user-images.githubusercontent.com/1099479/201077909-035579a2-889b-45bd-9c2d-b56d78074c28.gif)

maybe `&:not(:focus)` could help?